domain/lib/src/repository/sharedspace/shared_space_repository.dart

```diff
@@ -56,4 +56,6 @@ abstract class SharedSpaceRepository {
     SharedSpaceId sharedSpaceId,
     SharedSpaceRole sharedSpaceRole,
     EnableVersioningWorkGroupRequest enableVersioningWorkGroupRequest);
+
+  Future<List<SharedSpaceNodeNested>> getAllSharedSpaceDrive(SharedSpaceId sharedSpaceId);
```

Comment:
Please create `drive_repository` and `drive_datasource` for drive. Then move this function to `drive_repository`. After that, I think the method name should be `getAllWorkgroups(DrivedId driveId)`

Create `DriveId` also, then an extension `DrivedId SharedSpaceId.asDriveId()`